src/arrayMethodSort.js

```diff
@@ -3,9 +3,42 @@
 /**
  * Implement method Sort
  */
+
+const compareStrings = (previous, current) => {
+  const prev = previous.toString();
+  const curr = current.toString();
+
+  if (prev > curr) {
+    return 1;
+  }
+
+  if (prev === curr) {
+    return 0;
+  }
+
+  return -1;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(compareFunction = compareStrings) {
+    let sorted;
+
+    for (let j = this.length; !sorted; j--) {
+      sorted = true;
+
+      for (let i = 1; i < j; i++) {
+        const prev = this[i - 1];
+        const current = this[i];
+
+        if (compareFunction(prev, current) > 0) {
+          sorted = false;
```

Comment:
if on some iteration compare function return <= 0, then you will not return to the loop and stop sorting, maybe change condition of loop interrupting on 26 line to iterate by the whole array